src/config/config.ts

```diff
@@ -331,7 +337,12 @@ const config = convict({
 })
 
 // Perform validation
-config.validate({ allowed: "strict" })
+// TODO: remove try-catch after prod deployment is successful to avoid blocking
+try {
+  config.validate({ allowed: "strict" })
+} catch (e) {
+  console.log(e)
+}
```

Comment:
Do we intend to leave this in to catch any env vars we fail to add? If so, we might want to make the log statement easier to search e.g. `Convict error, missing env vars: ${e}`

Reply:
Ah that makes sense, let me update it

Reply:
done